src/tokens/functional/color/dark/patterns-dark.json5

```diff
@@ -1933,6 +1933,12 @@
       },
     },
   },
+  skeletonLoader: {
+    bgColor: {
+      $value: '{bgColor.muted}',
```

Comment:
Do we need to add an entry for dark mode if it's using the same token?

We use the same color between light and dark. We rely on `bgColor.muted` changing between dark and light modes.

Reply:
Yes, we need dark and light mode, just not any of the sub-themes.